_posts/2022-07-25-Building-CNF-Using-CNDP-Rust.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Building Cloud Native Network Functions using Rust
+author: CNDP Team
+excerpt_separator: <!--more-->
+---
+
+
+## Rust for Networking and Cloud Native Development

```

Comment:
Can you start with a deeper heading? The /blog page inserts the title as \<h2> so it looks weird if any of the headers in the post are \<h2> or bigger.